rich/crosshairs.py

```diff
@@ -0,0 +1,88 @@
+from typing import TYPE_CHECKING, Iterable
+
+
+from .jupyter import JupyterMixin
+from .measure import Measurement
+from .segment import Segment
+
+if TYPE_CHECKING:
+    from .console import Console, ConsoleOptions, RenderResult
+    from .style import StyleType
+
+
+class Crosshairs(JupyterMixin):
+    """Renders a crosshairs that spans the whole console window.
+
+    Args:
+        x (int): The x position of the cross centre.
+        y (int): The y position of the cross centre.
+        style (StyleType, optional): Style to render the crosshairs with. Defaults to "".
+    """
+
+    def __init__(self, x: int, y: int, style: "StyleType" = ""):
```

Comment:
Nitpick, I like to add `-> None:` on constructors for consistency (even though its kind of redundant).

Reply:
My bad, fixed.